<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
font: 16pt &quot;Bahnschrift&quot;;
background-color: rgb(239, 255, 239);
color: rgb(40, 70, 40);
}
QPushButton{
border-radius: 10px;
transition: .2s linear;
background-color: rgb(209, 250, 209);
padding:5px;
padding-left: 10px;
padding-right: 10px;
min-width: 140px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 25pt &quot;Bahnschrift&quot;;
</string>
        </property>
        <property name="text">
         <string>EyeSa</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="about_button">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>О приложении</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_button">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settings_button">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Настройки</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
